test/common_utils.py

```diff
@@ -283,6 +280,17 @@ def _check_module_exists(name):
 if TEST_NUMPY:
     import numpy
 
+ALL_TENSORTYPES = [torch.float,
+                   torch.double,
+                   torch.half]
+
+if TEST_WITH_ROCM:
```

Comment:
add a comment here saying that bfloat16 bringup is currently only available on ROCm - so that it's understood that this should eventually be unified again

Reply:
Okay.